generator/src/Definition/OperatorDefinition.php

```diff
@@ -63,6 +63,6 @@ public function __construct(
 
         $this->arguments = array_merge($requiredArgs, $optionalArgs);
 
-        $this->tests = array_map(static fn (array $test): TestDefinition => new TestDefinition(...$test), array_values($tests));
+        $this->tests = array_map(static fn (object $test): TestDefinition => new TestDefinition(...(array) $test), array_values($tests));
```

Comment:
I think this line of code is "expert" level.

Reply:
Splitting this line across multiple lines may just make it a little less "expert" and help tell the closure apart from the arg:
```suggestion
        $this->tests = array_map(
            static fn (object $test): TestDefinition => new TestDefinition(...(array) $test),
            array_values($tests),
        );
```